j/json-smart-v2-accessors-smart/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "maintainer": "pratibh-ibm",
+    "package_name": "json-smart-v2",
```

Comment:
Package name can be accessors-smart. Accordingly update wherever applicable like the folder name, build info, script name etc.